blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -1,32 +1,34 @@
-          ]]>
+                  return normalizedUrl;
+                }
+                var transformer = function(payload) {
+                  payload.data.request.user_ip = '0.0.0.0';
+                  payload.data.request.url = normalizeURL(window.location.pathname);
+                };
+                var _rollbarConfig = {
+                  accessToken: "81f3134dedf44871b9cc0a347b1313df",
+                  captureUncaught: true,
+                  code_version: "${it.blueOceanPluginVersion}",
+                  source_map_enabled: true,
+                  guess_uncaught_frames: true,
+                  transform: transformer
+                };
+                <![CDATA[
```

Comment:
@vivek sorry Vivek ... I didn't mean for you to wrap the `CDATA` section. That would normally wrap the whole thing.

Buuuttt .... this has drawn my attention to something I missed the first time I looked at this. What's inside that `CDATA` section? That looks like the minified version of some library ... Rollbar? Is [this NPM package](https://www.npmjs.com/package/rollbar) the same thing? If it is, then I definitely think we should pull this out of here and into a CommonJS module of its own and bundle it using the normal tools.